context.go

```diff
@@ -43,7 +43,7 @@ type Context struct {
 	// PrevHash is a hash of the previous block.
 	PrevHash util.Uint256
 
-	Timestamp uint32
+	Timestamp uint64
```

Comment:
We need to declare that this is a nanosecond-precision timestamp for every user of it to be aware.

Reply:
Added comment for Timestamp